spec/spec_helper.rb

```diff
@@ -1,6 +1,9 @@
 require 'simplecov'
 SimpleCov.start 'rails'
 
+# Required for subscriptions_controller tests
```

Comment:
This variable will be set to false in almost all the chatwoot installations, except for the possible one we will set up for the hosted version. Having the variable set here will affect the context for the entire codebase. Let's keep the assumption that the software should be tested on the context of this variable being set to 'false' . 

We only need to enable this variable in the spec/controllers/api/v1/subscriptions_controller_spec.rb.  So either lets assign the env variable at the beginning of the test. ( ensure its reset at the end of the test ) . or stub it for the specific test cases 

```
allow(ENV).to receive(:[]).with("BILLING_ENABLED").and_return(true)
```
ref: https://thoughtbot.com/blog/testing-and-environment-variables  ( for both examples )

Reply:
Hi, yes, I fixed the code climate issue. Sorry, I didn't notice it was on error.

About the ENV issue:
Like I commented in your first review, the climate_control gem and mocking the ENV doesn't work here because we need to set `"BILLING_ENABLED"` to true in the initialization of the test suite otherwise the subscription routes won't get loaded. Please have a look at `routes.rb:97`. The routes don't get initialized before every test.

Reply:
Also, could you retry the CI build? I don't think the error has something to do with my code changes :)

Reply:
@venarius It's something to do with the containers, we have fixed it on branches, I will rebase this once it is merged to develop

Reply:
@venarius  🤦‍♂️ apologies for missing out on the environment variable check in the routes file.  After looking at it again, I feel like having it in the `routes.rb`  to be a bad call.  let's remove that condition from the routes.  

and have a `before_action` in both `subscription_controller` and `webhook_controller` which throws 404 unless the variable is set.  Let me know if you want any help with the change ? 😃

Reply:
@sojan-official yes I will do that. No worries 😀